Makefile

```diff
@@ -0,0 +1,56 @@
+.PHONY: install
+install: ## Install the poetry environment and install the pre-commit hooks
+	@echo "🚀 Creating virtual environment using pyenv and poetry"
```

Comment:
I think emojis are not well handled by Haiku terminal currently, maybe they should be avoided

Reply:
Seems like not, it can be removed?

Reply:
Done